src/applications/mhv/medications/tests/e2e/pages/MedicationsListPage.js

```diff
@@ -191,5 +192,13 @@ class MedicationsListPage {
         'We got your request on October 4, 2023. Check back for updates.',
       );
   };
+
+  verifyNonVAPrescriptionNameOnListPage = () => {
+    cy.get(
+      `#card-header-${
+        nonVARx.data.id
+      } > [data-testid="medications-history-details-link"]`,
+    ).should('contain', 'CALAMINE');
```

Comment:
I would prefer a parameter is passed to this method.  So it has should('contain',name) as the assertion

Reply:
Fine to have to do for now.

Reply:
ok. making note of it and get it in one of my tests. Thanks!